### DISCUSSION TITLE
Best practice/optimal settings - DNS load balancing

### DISCUSSION THREAD

**Discussion by yolanjoy** [nginx/nginx#381] [Q&A]
Hello,

Can someone please advise on the best practice/optimal settings for DNS load balancing?

I've latest mainline nginx running.  I've been seeing multiple err/warn messages in the logs occasionally
. 
Thanks

**Comment by Maryna-f5** [comment]
This discussion appears to focus on the configuration, troubleshooting or general questions around running NGINX, which is not the intended use for GitHub discussions. 

We encourage you to visit [https://community.nginx.org ](https://community.nginx.org/)and share your discussion there as the NGINX Community Forum is the most appropriate place to engage for these types of questions.
